FakeServer/Program.cs

```diff
@@ -80,49 +61,51 @@ public static int Main(string[] args)
                .UseConfiguration(config)
                .UseStartup<Startup>()
                .UseSerilog();
-        
-        private static string GetAssemblyVersion()
-        {
-            return FileVersionInfo.GetVersionInfo(Assembly.GetExecutingAssembly().Location).ProductVersion;
-        }
 
-        private static Dictionary<string, string> ParseInMemoryCollection(string[] args)
+        private static CommandLineApplication BuildCommandLineApp(Func<string[], Dictionary<string, string>, int> invoke)
```

Comment:
Can you do a small code convention changes to this method, so it follows same convention as rest of the solution. Add some line changes before and after if-conditions etc. So something like this:

```cs
private static CommandLineApplication BuildCommandLineApp(Func<string[], Dictionary<string, string>, int> invoke)
        {
            var app = new CommandLineApplication(throwOnUnexpectedArg: false);
            app.HelpOption();

            var optionVersion = app.Option("--version", "Prints the version of the app", CommandOptionType.NoValue);
            var optionFile = app.Option<string>("--file <FILE>", "Data store's JSON file (default datastore.json)", CommandOptionType.SingleValue);
            var optionServe = app.Option("-s|--serve <PATH>", "Static files (default wwwroot)", CommandOptionType.SingleValue);

            app.OnExecute(() =>
            {
                if (optionVersion.HasValue())
                {
                    Console.WriteLine(GetAssemblyVersion());
                    return 0;
                }

                var initialData = new Dictionary<string, string>()
                {
                    { "file", optionFile.HasValue() ? optionFile.Value() : "datastore.json" }
                };

                initialData.TryAdd("currentPath", Directory.GetCurrentDirectory());

                if (optionServe.HasValue() && !string.IsNullOrEmpty(optionServe.Value()))
                {
                    if (!Directory.Exists(optionServe.Value()))
                    {
                        Console.WriteLine($"Folder doesn't exist: {optionServe.Value()}");
                        return 1;
                    }

                    initialData.Add("staticFolder", Path.GetFullPath(optionServe.Value()));
                    Console.WriteLine($"Static files: {initialData["staticFolder"]}");
                    // When user defines static files, fake server is used only to server static files
                }
                else
                {
                    Console.WriteLine($"Datastore file: {initialData["file"]}");
                    Console.WriteLine($"Datastore location: {initialData["currentPath"]}");
                    Console.WriteLine($"Static files: default wwwroot");
                }

                return invoke(app.RemainingArguments.ToArray(), initialData);
            });

            return app;
        }
```